pinot-controller/src/main/resources/app/interfaces/types.d.ts

```diff
@@ -119,6 +119,8 @@ declare module 'Models' {
     numGroupsLimitReached: boolean
     partialResponse?: number
     minConsumingFreshnessTimeMs: number
+    offlineThreadCpuTimeNs: number
+    realtimeThreadCpuTimeNs: number
```

Comment:
Don't follow why changes are needed in this and rest of the tsx, ts files?

Reply:
tsx, ts file change is needed because e need display those two metrics on UI.

Reply:
Got it. Thanks